templates/guest_book/gulpfilelocal.js

```diff
@@ -0,0 +1,120 @@
+const gulp = require("gulp");
+const fs = require('fs');
+const args = require("yargs").argv;
+const { Account, SimpleKeyStoreSigner, InMemoryKeyStore, KeyPair, LocalNodeConnection, NearClient, Near } = require('nearlib');
+
+gulp.task('copyfiles', function () {
+  return gulp.src('./assembly/**/*')
+    .pipe(gulp.dest('./out/'));
+});
+
+gulp.task("build:model", callback => {
+  const asc = require("assemblyscript/bin/asc");
+  asc.main([
+    "model.ts",
+    "--baseDir", "./out",
+    "--nearFile", "../out/model.near.ts",
+    "--measure"
+  ], callback);
+});
+
+gulp.task("build:bindings", ["build:model"], callback => {
+  const asc = require("assemblyscript/bin/asc");
+  asc.main([
+    "main.ts",
+    "--baseDir", "./out",
+    "--binaryFile", "../out/main.wasm",
+    "--nearFile", "../out/main.near.ts",
+    "--measure"
+  ], callback);
+});
+
+gulp.task("build:all", ["build:bindings"], callback => {
+  const asc = require("assemblyscript/bin/asc");
+  asc.main([
+    "../out/main.near.ts",
+    "--baseDir", "./out",
+    "-O3",
+    "--binaryFile", "../out/main.wasm",
+    "--sourceMap",
+    "--measure"
+  ], callback);
+});
+
+gulp.task('build', function(done) {
+  const runSequence = require('run-sequence');
+  runSequence('copyfiles', 'build:all', function() {
+    done();
+  });
+});
+
+gulp.task('deploy', function(done) {
+  // Find keys/ account id
```

Comment:
I think it makes sense to extract tooling into separate module (and later extract into separate `npm` package).

Reply:
yeah I agree. That's why it's a "first stab"